bindings/web/scripts/build.js

```diff
@@ -105,12 +105,27 @@ function ensure_rust_target_installed() {
   const targets = ret.stdout.toString(encoding = "ascii").trim().split("\n");
   if (targets.find((e) => e === TARGET) === undefined) {
     console.log(`Rust target \`${TARGET}\` is not installed !`)
-    console.log("Install it with:")
-    console.log(`  rustup target add ${TARGET}`)
-    exit(1);
+    install_rust_target(TARGET)
   }
 }
 
+function install_rust_target(target) {
+  const rustup_install_target_args = [
+    on_windows() ? "rustup.exe" : "rustup",
+    "target",
+    "install",
+    target
+  ];
+
+  console.log(`Installing rust target ${target}`);
```

Comment:
Maybe add some quotes as with the other messages
```suggestion
  console.log(`Installing rust target '${target}'`);
```